browser/brave_search/brave_search_browsertest.cc

```diff
@@ -72,33 +76,14 @@ class BraveSearchTest : public InProcessBrowserTest {
     command_line->AppendSwitch(switches::kIgnoreCertificateErrors);
   }
 
-  std::unique_ptr<net::test_server::HttpResponse> HandleRequest(
```

Comment:
why did you change this part? https://github.com/brave/brave-core/pull/8537/files#diff-b778bb12795e38ebfc143f5ab4608badd1d17d829b55742e138ae0c942b3af27R42 seems like it is better suited for unit tests. For browser tests we should be handling the response from the test server or we're bypassing really important parts of the code we need to test

Reply:
reverted it